src/Charcoal/Queue/QueueItemInterface.php

```diff
@@ -9,6 +9,10 @@
  */
 interface QueueItemInterface extends ModelInterface
 {
+    const SUCCESS_STATUS = 'success';
+    const FAILED_STATUS = 'failed';
+    const RETRY_STATUS = 'retry';
```

Comment:
I would be inclined to use a prefix instead of suffix and assign the constant as the value to reduce conflicts with any future constants:

```suggestion
    const STATUS_SUCCESS = 'STATUS_SUCCESS';
    const STATUS_FAILED  = 'STATUS_FAILED';
    const STATUS_RETRY   = 'STATUS_RETRY';
```

Maybe we could ask for a consensus from the team.

Reply:
Done